src/main/java/com/superdevs/HealthOMeter/entity/Contact.java

```diff
@@ -0,0 +1,232 @@
+package com.superdevs.HealthOMeter.entity;
+
+import javax.validation.constraints.NotBlank;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity(name = "contacts")
+public class Contact {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private long id;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "contacts")
+    private Set<BioMetricData> bioMetricData;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "contacts")
+    private Set<BioMetricHistoricalData> bioMetricHistoricalData;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "contacts")
+    private Set<BMIData> bmiData;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "contacts")
+    private Set<WHRData> whrData;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "contacts")
+    private Set<RFMData> rfmData;
+
+    @ManyToMany
+    @JoinTable(
+            name = "historicalcalculatorsresults_contact",
+            joinColumns = @JoinColumn(name = "contact_id"),
+            inverseJoinColumns = @JoinColumn(name = "historicalcalculatorsresults_id")    )
+    private Set<HistoricalCalculatorsResults> historicalCalculatorsResults;
```

Comment:
I've got in here a little doubt.

So - the way are joinColums and inverseJoin colums made shouldn't look like this (example below)?

 @JoinTable(
        name = "Employee_Project", 
        joinColumns = { @JoinColumn(name = "employee_id") }, 
        inverseJoinColumns = { @JoinColumn(name = "project_id") }
    )

Reply:
> I've got in here a little doubt.
> 
> So - the way are joinColums and inverseJoin colums made shouldn't look like this (example below)?
> 
> @JoinTable(
> name = "Employee_Project",
> joinColumns = { @JoinColumn(name = "employee_id") },
> inverseJoinColumns = { @JoinColumn(name = "project_id") }
> )

No Many@many at all -pointless discussion